native/src/main/java/io/ballerina/stdlib/xslt/XsltTransformer.java

```diff
@@ -52,11 +53,13 @@
 public class XsltTransformer {
 
     private static final Logger log = LoggerFactory.getLogger(XsltTransformer.class);
+    private static final PrintStream errStream = System.err;
     private static final String OPERATION = "Failed to perform XSL transformation: ";
 
     public static Object transform(BXml xmlInput, BXml xslInput) {
 
         try {
+            //System.setProperty("javax.xml.transform.TransformerFactory", "net.sf.saxon.BasicTransformerFactory");
```

Comment:
Let's remove commented lines

Reply:
Updated in [cfe19f6](https://github.com/ballerina-platform/module-ballerina-xslt/pull/427/commits/cfe19f6c1557bd6f2848b10770c41649fc72e896)